gooey/gui/processor.py

```diff
@@ -59,16 +61,59 @@ def run(self, command):
         t = Thread(target=self._forward_stdout, args=(self._process,))
         t.start()
 
+    def format_interval(self,time_value):

```

Comment:
`Processor` is now doing a bit too much imo (it shouldn't have dealt with the finer details of progress here either, to be fair). You should pull out the time stuff into a diff module. It'll make your tests a bit clearer, too, as you won't have to call through `CommandProcessor` just to test that something gets formatted correctly.

Reply:
Separated into new module gooey/gui/util/time.